library/src/interactor_impl.cxx

```diff
@@ -260,10 +262,14 @@ class interactor_impl::internals
       self->Interactor.SetCommandBuffer(commandWithArgs);
     }
     else if (event == vtkF3DConsoleOutputWindow::ShowEvent ||
-      event == vtkF3DConsoleOutputWindow::HideEvent)
+      event == vtkF3DConsoleOutputWindow::HideConsoleEvent)
     {
       self->Options.ui.console = (event == vtkF3DConsoleOutputWindow::ShowEvent);
     }
+    else if (event == vtkF3DConsoleOutputWindow::HideMinimalConsoleEvent)
```

Comment:
I dont think we need a dedicated event, since we can only show one console, you can just hide all console on hide event